package.json

```diff
@@ -20,6 +20,7 @@
     "@angular/platform-browser": "~13.1.0",
     "@angular/platform-browser-dynamic": "~13.1.0",
     "@angular/router": "~13.1.0",
+    "lodash": "^4.17.21",
```

Comment:
On va encore devoir faire un npm i !!!!

Reply:
Certainement...

Reply:
yes, first thing to do after doing a pull :)